homeassistant/helpers/restore_state.py

```diff
@@ -18,7 +17,6 @@
 from .entity import Entity
 from .event import async_track_time_interval
 from .json import JSONEncoder
-from .singleton import singleton
 from .storage import Store
 
 DATA_RESTORE_STATE_TASK = "restore_state_task"
```

Comment:
Let’s rename this constant since it now stores the actual data.

Reply:
Sounds good. Will drop `_task`